addons/website_event_track_quiz/models/event_track_visitor.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class TrackVisitor(models.Model):
+    _name = 'event.track.visitor'
+    _inherit = ['event.track.visitor']
+
+    quiz_completed = fields.Boolean('Completed')
+    points_gained = fields.Integer("Points gained on this quiz", default=0)
```

Comment:
quiz_points maybe ? not sure gained is really useful, and are we sure it is english-enough :D ?